op-dispute-mon/mon/service.go

```diff
@@ -89,6 +90,9 @@ func (s *Service) initFromConfig(ctx context.Context, cfg *config.Config) error
 	if err := s.initOutputRollupClient(ctx, cfg); err != nil {
```

Comment:
`initOutputRollupClient` still expects a configured `RollupRpc`. Since the rollup-rpc flag is now optional, it should be tweaked to not fail if the flag isn't set.